app.py

```diff
@@ -6,7 +6,17 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session.get_items()
+    session.add_item('Avi')
+    return render_template("index.html", items=items)
+
+
+@app.route('/', methods=['POST'])
+def create():
+    items = session.get_items()
+    session.add_item(request.form.get('item_title'))
+    return render_template("index.html", items=items)
```

Comment:
Instead of making this path render the index page, you could instead redirect the user to the home page. e.g. `return redirect("/")`

That will simplify this line and if index() changes later, you won't need to update create()